src/AML/Notebooks/ModelTraining.ipynb

```diff
@@ -63,35 +63,6 @@
    "source": [
     "## Data preparation\n",
     "\n",
-    "### Elimination of possible false negatives\n",
-    "\n",
-    "For each sequence not leading up to a failure, we remove the \"tail\" of the length equal the future horizon to eliminate the possibility of training and testing the model on false negatives."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 3,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "seed = 42\n",
-    "\n",
-    "max_cycles = combined_df.join(\n",
-    "    combined_df[(combined_df.rul.isna())]\n",
-    "    .groupby(['machineID'], as_index=False).cycle\n",
-    "    .agg(['max', 'count']).apply(\n",
-    "        lambda x: x['max'] - min(x['count'], w), axis = 1)\n",
-    "    .to_frame(name='max_cycle'), on='machineID')\n",
-    "\n",
-    "data = max_cycles[max_cycles.max_cycle.isna() |\n",
```

Comment:
You may need to do a little more than simply removing it. Cells below reference the data variable which is set here. Run the notebooks to test. Also, does the removal of this code affect model's accuracy in any way?